native/src/main/java/io/ballerinax/salesforce/CSVParserUtil.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerinax.salesforce;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvException;
+import io.ballerina.runtime.api.creators.ErrorCreator;
+import io.ballerina.runtime.api.creators.TypeCreator;
+import io.ballerina.runtime.api.creators.ValueCreator;
+import io.ballerina.runtime.api.types.ArrayType;
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.values.BArray;
+import io.ballerina.runtime.api.values.BString;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.List;
+
+/**
+ * This class holds the utility methods involved with parsing csv content.
+ *
+ * @since 8.0.1
```

Comment:
Is this correct?

Reply:
Why is it not?

> The `@since` tag should indicate the version that will include the new class when it is released. If version 1.2.1 is the upcoming release that will introduce the new class, you would use `@since 1.2.1` to document it. 

This is what I found when searched.

Reply:
My bad, I thought the salesforce latest version was 8.0.1 and this is going for 8.0.2